babel.config.js

```diff
@@ -78,10 +78,18 @@ module.exports = {
       ],
     },
     esm: {
-      plugins: [...productionPlugins, ['@babel/plugin-transform-runtime', { useESModules: true }]],
+      plugins: [
+        ...productionPlugins,
+        ['@babel/plugin-transform-runtime', { useESModules: true }],
+        ['./packages/babel-plugin-material-ui', { useESM: true }],
```

Comment:
I don't fully understand, why using the Babel plugin here?

Reply:
Because the es/esm version of `@material-ui/core` doesn't import the es/esm version of `@material-ui/styles`.
Same goes for the other packages.

https://unpkg.com/browse/@material-ui/core@4.3.3/esm/styles/makeStyles.js

Reply:
Reverting for the same reason as https://github.com/mui-org/material-ui/pull/17015/files#r319671043